idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/settings/tabs/export/IndentSizeSection.kt

```diff
@@ -0,0 +1,148 @@
+package io.github.composegears.valkyrie.ui.screen.settings.tabs.export
+
+import androidx.compose.animation.AnimatedContent
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.scaleIn
+import androidx.compose.animation.scaleOut
+import androidx.compose.animation.togetherWith
+import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.github.composegears.valkyrie.ui.foundation.CenterRow
+import io.github.composegears.valkyrie.ui.foundation.dim
+import io.github.composegears.valkyrie.ui.foundation.theme.PreviewTheme
+
+@Composable
+fun IndentSizeSection(
+    indent: Int,
+    onValueChange: (Int) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var expanded by remember { mutableStateOf(false) }
+    val rotateAnimation = remember { Animatable(0f) }
+    ListItem(
+        modifier = modifier.padding(horizontal = 8.dp),
```

Comment:
i suggest make `ListItem` clickable to make this behavior

```kotlin
modifier = modifier
            .clickable { expanded = true }
            .padding(horizontal = 8.dp)
```

https://github.com/user-attachments/assets/6797a63b-f5d4-41b7-ab1d-d106feed1ea2